### PR TITLE
Added a custom keymap for XD60

### PR DESCRIPTION
**Pull request by suryanisaac**
## Description

Added a custom XD60 keymap, for anyone interested it trying it out. Based on my own trial and error to find the best nav postitions. Most important keys on or close to the home row.

More information in README.md
 
Flashed and tested it on my XD60 Rev 3 and it's working fine.

### CODE REVIEW

File: keyboards/xd60/keymaps/suryanisaac/keymap.c [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/xd60/keymaps/suryanisaac/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/xd60/keymaps/suryanisaac/keymap.c (hunk 1 above)
Comment (suryanisaac) [reply keyboards/xd60/keymaps/suryanisaac/keymap.c:1]: Added that. Is there anything else I need to do?
Thanks

Comment (drashna) [reply keyboards/xd60/keymaps/suryanisaac/keymap.c:17]: Nope, at least, not that I saw.

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Though, would you mind removing the extra json files from the folder?

**Comment by suryanisaac** [comment]
Done

**Review by tzarc — APPROVED** [review]

**Pull request merged**